src/main/java/characterpicker/infrastructure/CompletionWeightCalculator.java

```diff
@@ -0,0 +1,56 @@
+package characterpicker.infrastructure;
+
+import java.util.List;
+
+public class CompletionWeightCalculator {
+
+    private CompletionWeightCalculator(){
+        throw new IllegalStateException("Utility class");
+    }
+    //TO DO
+    //change List name
+     static Integer calculateCompletionWeight(List<String> list) {
+
+        double sumOfWeights = 0;
+
+        for (String boss : list) {
+            if (boss.contains("mom's hearth")) {
```

Comment:
`equals` zamiast contains, bo contains zadziała nawet jak będzie `adawda wd mom's hearthdawdwad`

Reply:
I brakuje na to testu też w takim razie

Reply:
I brakuje testu (i co za tym idzie obsługi) tego co się stanie jak ktoś poda coś nie z listy.

Reply:
Nawet lepszy przykład jak będziesz miał `cain` i `teinted cain` w tym kodzie tutaj, to jak ktoś poda `cain` to policzy mu punkty za obie wersje.